modules/View.js

```diff
@@ -56,6 +56,70 @@
       if (!options.noActivate) {
         this.activate();
       }
+      this.updateDelegateEvents();
+    },
+
+
+    /**
+    * @param method {function} callback method of event
+    * @param eventName {String} description of event type 
+    * @return modified mehtod with before/after signals sent to Logger
+    * @method trackEvents
+    */
+    trackEvents : function(method, eventName){
+      var self = this;
+      var methodCopy = method;
+      method = _.bind(function(){
+        var UUID = "uuid-"+(new Date()).getTime().toString(16)+Math.floor(1E7*Math.random()).toString(16);
+        Logger.track({
+          UUID : UUID,
+          type : "clickEvent",
+          state: "start",
+          eventName: eventName,
+          time: Date.now(),
+        });
+        methodCopy.call(self);
+        Logger.track({
+          UUID: UUID,
+          time: Date.now(),
+          state: "end",
+        });
+      },this);
+      return method;
+      },
+
+    /**
+    * updates Backbone's delegateEvents to bind signals to Logger to all callback 
+    * functions in the View's list of events
+    * @method updateDelegateEvents
+    */
+    updateDelegateEvents: function(){
+      var backboneDelegateEvents = Backbone.View.prototype.delegateEvents;
+      Backbone.View.prototype.delegateEvents = function(events){
+        var delegateEventSplitter = /^(\S+)\s*(.*)$/;
```

Comment:
In delegateEvents, I'm modifying the callback functions that are assigned to certain event triggers. delegateEvents is only called after every render to tie user actions to callback functions, but the function themselves are not called by delegate events, they're fired in response to clicks or key presses. Wrapping delegateEvents won't give us information about the timing of events that happen in response to clicks/key presses. To track these events, I wanted to wrap the callback functions with before/after signals sent to logger. Is there a way I can do that without copying delegateEvents from backbone?